packages/accounts/src/kernel-zerodev/account.ts

```diff
@@ -67,6 +70,19 @@ export class KernelSmartContractAccount<
     return this.encodeExecuteAction(target, value, data, 1);
   }
 
+  override async encodeBatchExecute(
+    _txs: BatchUserOperationCallData
+  ): Promise<`0x${string}`> {
+    // @ts-ignore
+    const multiSendData: `0x${string}` =
+      "0x" + _txs.map((tx) => encodeCall(tx)).join("");
```

Comment:
Btw you could use the `concat` util from viem and just do 

```
concat(_txs.map(...))
```

Reply:
done